src/gh_stats/cli.py

```diff
@@ -190,6 +194,7 @@ def main(repos, org, user, output_file, auth_token):
                     str(data["Clones Unique"]),
                     str(data["Views Total"]),
                     str(data["Views Unique"]),
+                    str(period.capitalize()),
```

Comment:
As not all stats on the row are in the same period, I prefer this version: 
```python
            table = Table(title="GitHub Stats")
            table.add_column("Repo")
            table.add_column("Forks", justify="center")
            table.add_column("Stars", justify="center")
            table.add_column("Watchers", justify="center")
            table.add_column(f"Clones Total\n per {period.capitalize()}", justify="center")
            table.add_column(f"Clones Unique\n per {period.capitalize()}", justify="center")
            table.add_column(f"Views Total\n per {period.capitalize()}", justify="center")
            table.add_column(f"Views Unique\n per {period.capitalize()}", justify="center")

            for data in final_data:
                table.add_row(
                    str(data["Repo"]),
                    str(data["Forks"]),
                    str(data["Stars"]),
                    str(data["Watchers"]),
                    str(data["Clones Total"]),
                    str(data["Clones Unique"]),
                    str(data["Views Total"]),
                    str(data["Views Unique"]),
                )
```

What are your thoughts?

Reply:
Yeah, that sounds good. I'll test it out on my end and see how it looks first.